### PR TITLE
[KOGITO-2317] - Unify Kogito Quarkus and Spring Images

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-2317

Links to related PRs:
https://github.com/kiegroup/kogito-images/pull/348
https://github.com/kiegroup/kogito-cloud-operator/pull/743

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by radtriste** [comment]
This PR can only be merged just before the next release, else it will break all future PR checks until the new Kogito images are released

File: serverless-workflow-github-showcase/pr-checker-workflow/README.md [hunk 1]
```diff
@@ -101,11 +101,12 @@ You should see a similar output like this:
...
-STEP 1: FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
-STEP 2: COPY target/*-runner.jar $KOGITO_HOME/bin
+STEP 1: FROM quay.io/kiegroup/kogito-runtime-jvm:latest
+STEP 2: ENV RUNTIME_TYPE quarkus
+STEP 3: COPY target/*-runner.jar $KOGITO_HOME/bin
 --> 58760d128d8
-STEP 3: COPY target/lib $KOGITO_HOME/bin/lib
-STEP 4: COMMIT quay.io/ricardozanini/pr-checker-workflow:latest
+STEP 4: COPY target/lib $KOGITO_HOME/bin/lib
+STEP 5: COMMIT quay.io/ricardozanini/pr-checker-workflow:latest
```
Comment (ricardozanini) [inline serverless-workflow-github-showcase/pr-checker-workflow/README.md]: Ops, can you please change this namespace to something like `example`?

**Review by ricardozanini — APPROVED** [review]

File: serverless-workflow-github-showcase/pr-checker-workflow/README.md (hunk 1 above)
Comment (Kaitou786) [reply serverless-workflow-github-showcase/pr-checker-workflow/README.md]: remove from other places as well :p

**Review by ricardozanini — APPROVED** [review]

**Comment by radtriste** [comment]
Jenkins retest this

**Comment by radtriste** [comment]
Jenkins retest this

**Comment by radtriste** [comment]
Jenkins retest this

**Comment by spolti** [comment]
```
Step 1/4 : FROM quay.io/kiegroup/kogito-runtime-jvm:latest
unauthorized: access to the requested resource is not authorized
```

```
docker pull quay.io/kiegroup/kogito-runtime-jvm:latest
Error response from daemon: repository quay.io/kiegroup/kogito-runtime-jvm not found: name unknown: repository name not known to registry
```
Shouldn't it be pointing to the nightly?
@radtriste @Kaitou786 ^

**Comment by Kaitou786** [comment]
We have always kept the latest released version of the image here, but since the images haven't been released so the test fails. 
But I kind of agree we can keep the nightly on the master and maybe the latest released on the stable branch(talking about all the images we use not just these new ones)
Wdyt @radtriste @spolti ?

**Comment by Kaitou786** [comment]
Jenkins retest this

**Comment by ricardozanini** [comment]
@radtriste, once you get back, you can merge if everything is alright.

**Comment by radtriste** [comment]
@Kaitou786 Agree that we could use the nightly images on master, even if we have a cyclic dependency here ...
images depending on examples which depend on images (for Dockerfiles) ... we need to find a way to correct this ...

**Comment by radtriste** [comment]
raised https://kie.zulipchat.com/#narrow/stream/232676-kogito/topic/Examples.20testing.20using.20Kogito.20images.20bring.20cyclic.20dependency and https://issues.redhat.com/browse/KOGITO-4609

**Review by evacchi — APPROVED** [review]

**Pull request merged**